python/upt-macports/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                upt-macports
+version             0.2.1
+revision            0
+
+maintainers         nomaintainer
+description         MacPorts backend for upt.
+long_description    ${description}
+
+platforms           darwin
+supported_archs     noarch
+
+homepage            https://github.com/mojca/upt-macports
```

Comment:
That's the wrong URL, it's just a temporary repository where I uploaded what I had so far, and it doesn't contain all the required changes you would need to make it functional.

I would move the package to either `macports-gsoc` or `macports` directly. I don't have sufficient permissions to create a repository under macports directly, but maybe @jmroot or @ryandesign could help? We could also leave it under `macports-gsoc/upt-macports` for now.

Reply:
What URL should I keep until a repo is made ?